planutils/packages/TEMPLATE/manifest.json

```diff
@@ -1,6 +1,45 @@
 {
-    "name": "Full name of package",
```

Comment:
Ok to include some of the executable manifest here, but best not to use macq for it.